src/Composition/joints/object3DMotion.jl

```diff
@@ -49,7 +50,8 @@ mutable struct FreeMotion <: Modia3D.AbstractJoint
                           r::AbstractVector   = Modia3D.ZeroVector3D,
                           rot::AbstractVector = Modia3D.ZeroVector3D,
                           v::AbstractVector   = Modia3D.ZeroVector3D,
-                          w::AbstractVector   = Modia3D.ZeroVector3D)
+                          w::AbstractVector   = Modia3D.ZeroVector3D,
+                          next_isrot123::Bool = true)  # dummy argument, is ignored
```

Comment:
I don't understand `next_isrot123` here. You don't use it in the constructor.

Reply:
Implemented by @MartinOtter.